src/Runtime/ExecutionSession.hpp

```diff
@@ -24,15 +24,15 @@
 
 namespace onnx_mlir {
 
-typedef OMTensorList *(*entryPointFuncType)(OMTensorList *);
-typedef const char *(*signatureFuncType)();
-
+using entryPointFuncType = OMTensorList *(*)(OMTensorList *);
+using queryEntryPointsFuncType = const char **(*)();
+using signatureFuncType = const char *(*)(const char *);
 using OMTensorUniquePtr = std::unique_ptr<OMTensor, decltype(&omTensorDestroy)>;
 
 class ExecutionSession {
 public:
   ExecutionSession(std::string sharedLibPath);
-  ExecutionSession(std::string sharedLibPath, std::string entryPointName);
+  ExecutionSession(std::string sharedLibPath, bool defaultEntryPoint);
```

Comment:
nit, why not just one `ExecutionSession(std::string sharedLibPath, bool defaultEntryPoint=true);`

Reply:
Done. Now we use a single constructor.